components/brave_wallet_ui/common/slices/entities/token-balance.entity.ts

```diff
@@ -2,7 +2,12 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
-
 export interface TokenBalancesForChainId {
   [contractAddress: string]: string
 }
+
+export type TokenBalancesRegistry = {
+  [address: string]: {
+    [chainId: string]: TokenBalancesForChainId
+  }
+}
```

Comment:
The naming seems a bit misleading. WDYT about this:

1. `TokenBalances` / `TokenBalancesForChainId`
2. `TokenBalancesForChainIds`
3. `TokenBalancesForAddressesAndChainIds` / `TokenBalancesRegistry`

By the way, I prefer keeping `TokenBalancesRegistry` because of the `tokenBalancesRegistry` variable used everywhere.